crates/cheatcodes/spec/src/vm.rs

```diff
@@ -1522,6 +1522,10 @@ interface Vm {
         string calldata error
     ) external pure;
 
+    /// Returns true if the left string contains the right string, otherwise returns false.
+    #[cheatcode(group = Testing, safety = Safe)]
+    function contains(string calldata left, string calldata right) external returns (bool result);
```

Comment:
Matching the convention used in solady which makes the UX clear: https://github.com/Vectorized/solady/blob/502cc1ea718e6fa73b380635ee0868b0740595f0/src/utils/LibString.sol#L611
```suggestion
    function contains(string calldata subject, string calldata search) external returns (bool result);
```